README.md

```diff
@@ -208,6 +208,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Bruzu](https://bruzu.com/) — Automate Image production. Generate tons of Image variants with API, Integrations or nocode sheet. API is FREE with watermark.
   * [Calendarific](https://calendarific.com) - Enterprise-grade Public holiday API service for over 200 countries. Free plan includes 1000 calls per month.
   * [Clarifai](https://www.clarifai.com) — Image API for custom face recognition and detection. Able to train AI models. Free plan has 5000 calls per month.
+  * [cloudlayer.io](https://cloudlayer.io) - PDF Generation and Image Generation service used to automate your content generation workflow. Free plan includes 100 documents for life, never expires.
```

Comment:
Sorry I think a 100 documents for life isnt really what we had in mind with a free tier, unless that means a 100 documents a month every month for life.

Reply:
That is unfortunate that is still confusing, one good thing is your helping me fine tune the verbiage on our page to make it more clear.  I will submit another PR once we adjust it.  Thanks!